client/packages/core/src/Reactor.js

```diff
@@ -1817,7 +1863,27 @@ export default class Reactor {
     }
   }
 
+  async syncUserToEndpoint(user) {
+    if (this.config.cookieEndpoint) {
+      try {
+        fetch(this.config.cookieEndpoint + '/sync-auth', {
```

Comment:
nit: if this is called cookieEndpoint, imo we should be probbly include the `sync-auth` in it. 

Reasoning: a user _may_ want to have a specific name for their endpoint.